_po/ja/how-to/laravel.po

```diff
@@ -768,21 +1175,33 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"        for($i =0; $i<2000; $i++)\n"
+"        {\n"
+"            \\App\\Models\\Post::factory()->count(100)->create();\n"
+"        }\n"
+"    }\n"
+"}\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.